### PR TITLE
Custom preflight class for tailwind use

### PR DESCRIPTION
**Pull request by KievDevel**

### CODE REVIEW

File: src/App/preflight.scss [hunk 1]
```diff
@@ -0,0 +1,391 @@
...
+    }
+
+    /* Make elements with the HTML hidden attribute stay hidden by default */
+    [hidden] {
+        display: none;
+    }
+
+    p {
+        font-size: 20px;
+    }
```
Comment (PascalPickel) [inline src/App/preflight.scss:390]: I cannot find this in tailwind base styles?
Where is this coming from?

File: src/App/preflight.scss [hunk 2]
```diff
@@ -0,0 +1,391 @@
...
+    ul,
+    menu {
+        list-style: none;
+        margin: 0;
+        padding: 0;
+    }
+
+    dialog {
+        padding: 0;
+    }
```
Comment (PascalPickel) [inline src/App/preflight.scss:318]: I cannot find this in tailwind base styles?
Where is this coming from?

File: src/App/preflight.scss [hunk 3]
```diff
@@ -0,0 +1,391 @@
...
+    */
+
+    button,
+    input,
+    optgroup,
+    select,
+    textarea {
+        font-family: inherit; /* 1 */
+        font-feature-settings: inherit; /* 1 */
+        font-variation-settings: inherit; /* 1 */
```
Comment (PascalPickel) [inline src/App/preflight.scss:178]: I cannot find this in tailwind base styles?
Where is this coming from?

File: src/App/preflight.scss [hunk 4]
```diff
@@ -0,0 +1,391 @@
...
+    /*
+    Add the correct display in Chrome and Safari.
+    */
+
+    summary {
+        display: list-item;
+    }
+
+    /*
+    Removes the default spacing for appropriate elements.
```
Comment (PascalPickel) [inline src/App/preflight.scss:280]:
```suggestion
    Removes the default spacing and border for appropriate elements.
```
This was the original comment I found in the base style linked by tailwind.

**Review by PascalPickel — COMMENTED** [review]
Found some differences based on the the base style linked by the tailwind docs:
https://unpkg.com/tailwindcss@3.3.2/src/css/preflight.css

File: src/App/preflight.scss (hunk 1 above)
Comment (KievDevel) [reply src/App/preflight.scss:390]: Thanks for noticing, this one comes from testing, thought I did remove it.

File: src/App/preflight.scss (hunk 4 above)
Comment (KievDevel) [reply src/App/preflight.scss:280]: This comes from here https://github.com/tailwindlabs/tailwindcss/blob/2bfdbce964f36b57fbac1725c807805f1d7a6ad7/src/css/preflight.css#L269
...and I do not see any border styles below

Comment (PascalPickel) [reply src/App/preflight.scss:280]: Ah, I was looking at the preflight linked to from the documentation. Not sure why they are different or if it matters which one we use?

Comment (KievDevel) [reply src/App/preflight.scss:280]: I believe github repo version is just up to date. Shouldn't be a big diff for us.

**Review by PascalPickel — APPROVED** [review]

**Pull request merged**